src/Dapr.Actors/Helper.cs

```diff
@@ -26,5 +27,19 @@ public static string GetCallContext()
                 return Guid.NewGuid().ToString();
             }
         }
+
```

Comment:
Is `GetCallContext()` ^^^ used for anything after your changes? I have a feeling this is vestigial and can be removed.

Reply:
It's referenced, but I'm not really sure what it's accomplishing. I can make an issue to remove it if you'd like, but would prefer to not have that in this PR.